msteams-platform/apps-in-teams-meetings/build-tabs-for-meeting.md

```diff
@@ -176,15 +176,125 @@ The Microsoft Teams JavaScript client SDK is a rich SDK used to create Tabs usin
 
 Microsoft Teams JavaScript library exposes the frameContext in which your meeting tab URL is loaded in the getContext API. The possible values of frameContext are content, task, setting, remove, sidePanel, and meetingStage. This allows you to build customized experiences based on where the app renders. For example, showing a specific collaboration focused UI when in the `meetingStage` and a different meeting preparation UI in the chat tab (`content`). For more information, see [getContext API](/microsoftteams/platform/tabs/how-to/access-teams-context?tabs=teamsjs-v2).
 
+## App caching
+
+App caching helps you to improve subsequent launch time of the apps that are loaded in the meeting side panel.
+
+> [!NOTE]
+>
+> * Currently, app caching is available only in [public developer preview](~/resources/dev-preview/developer-preview-intro.md).
+> * App caching is supported only for tabs loaded in the meeting side panel in Teams desktop client.
+> * Currently, app caching is not supported on chats, channels, and personal apps.
+
+### Enable app caching
+
+To enable app caching in your meeting side panel, follow the steps:
+
+1. Call `microsoftTeams.registerBeforeUnloadHandler` and `microsoftTeams.registerOnLoadHandler` (Work in Progress).
+
+1. Use `contentUrl` and `entityId` into the load handler to route to the correct page within your app and invoke `notifySuccess/notifyFailure` to notify Teams client that the app initialization flow is complete.
+
+1. Dispose resources and perform any cleanup needed in the `beforeUnload handler`, then invoke the `readyToUnload` callback to notify Teams client that the app unload flow is complete.
+
+The following is the flow diagram of the app added to the meeting stage without app caching:
+
+:::image type="content" source="../assets/images/saas-offer/first-launch-app.png" alt-text="This screenshot shows the flow of the first launch of the app in meeting stage.":::
+
+The following is the flow diagram of a cached app when it's added to the meeting stage:
+
+:::image type="content" source="../assets/images/saas-offer/cached-launch-app.png" alt-text="This screenshot shows the flow of the cached launch of the app in meeting stage.":::
+
+After you enable app caching, the webview that is used to host the embedded app is reused as users navigate to different instances of the app within a window.
+
+The webview of the app remains in the Document Object Model (DOM). The webview is hidden when the users go out of the app and shown when the users return to the app. When the app is cached, any audio that is playing is muted.
```

Comment:
Rephrase as:  Remove the first line about the DOM. The webview used to host the App is hidden when users....

Reply:
Currently we are not removing the step about DOM here, as there are other instances about DOM mentioned in the document. Removing this instance here would make other instances irrelevant.

<img width="443" alt="image" src="https://user-images.githubusercontent.com/95284144/201065654-6d83a44b-8cc7-4ba5-b1e8-606e138bd886.png">


Kindly suggest if we have to removed DOM in all the instances or we can continue with this?